### PR TITLE
Release notes for 7.13.3

### PR DESCRIPTION
**Pull request by logstashmachine**

### CODE REVIEW

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -46,6 +47,32 @@ This section summarizes the changes in the following releases:
...
+*Xml Filter - 4.1.2*
+
+* [DOC] Updated docs to correct name of parse_options config option https://github.com/logstash-plugins/logstash-filter-xml/pull/75[#75]
+
+*Beats Input - 6.1.5*
+
+* Changed jar dependencies to reflect newer versions https://github.com/logstash-plugins/logstash-input-http/pull/425[#425]
+* Fix: reduce error logging on connection resets https://github.com/logstash-plugins/logstash-input-beats/pull/424[#424]
+
+
```
Comment (jsvd) [inline docs/static/releasenotes.asciidoc:75]:
```suggestion

```

File: docs/static/releasenotes.asciidoc [hunk 2]
```diff
@@ -46,6 +47,30 @@ This section summarizes the changes in the following releases:
...
+* Fix: avoid long thread sleeps on codec close https://github.com/logstash-plugins/logstash-codec-multiline/pull/67[#67]
+
+*Xml Filter - 4.1.2*
+
+* [DOC] Updated docs to correct name of parse_options config option https://github.com/logstash-plugins/logstash-filter-xml/pull/75[#75]
+
+*Beats Input - 6.1.5*
+
+* Changed jar dependencies to reflect newer versions https://github.com/logstash-plugins/logstash-input-http/pull/425[#425]
+* Fix: reduce error logging on connection resets https://github.com/logstash-plugins/logstash-input-beats/pull/424[#424]
```
Comment (jsvd) [inline docs/static/releasenotes.asciidoc:73]:
```suggestion
* Fix: reduce error logging on connection resets https://github.com/logstash-plugins/logstash-input-beats/pull/424[#424]

```

File: docs/static/releasenotes.asciidoc [hunk 3]
```diff
@@ -46,6 +47,31 @@ This section summarizes the changes in the following releases:
...
+
+* Fix: avoid long thread sleeps on codec close https://github.com/logstash-plugins/logstash-codec-multiline/pull/67[#67]
+
+*Xml Filter - 4.1.2*
+
+* [DOC] Updated docs to correct name of parse_options config option https://github.com/logstash-plugins/logstash-filter-xml/pull/75[#75]
+
+*Beats Input - 6.1.5*
+
+* Changed jar dependencies to reflect newer versions https://github.com/logstash-plugins/logstash-input-http/pull/425[#425]
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc:72]:
```suggestion
* Changed jar dependencies to reflect newer versions https://github.com/logstash-plugins/logstash-input-beats/pull/425[#425]
```

**Review by karenzone — CHANGES_REQUESTED** [review]
Needs this fix to unresolvable link.

**Comment by karenzone** [comment]
Link fixed in plugin repo: https://github.com/logstash-plugins/logstash-input-beats/pull/427

**Review by karenzone — APPROVED** [review]
Builds cleanly, links resolve correctly. LGTM

**Pull request merged**